recipes/nncf/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "nncf" %}
+{% set version = "2.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8bdfabaf1b163da6e80dc2d7906c2c8342567cff28c40e97c2cb65929a750736
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - addict >=2.4.0
+    - jsonschema ==3.2.0
+    - jstyleson >=0.0.2
+    - matplotlib-base >=3.3.4  # [py>=37]
+    - natsort >=7.1.0
+    - networkx >=2.5
+    - ninja >=1.10.0.post2
+    - numpy ~=1.19.2
+    - pandas >=1.1.5  # [py>=37]
+    - pydot >=1.4.1
+    - python
+    - scikit-learn >=0.24.0
+    - scipy <=1.5.4,>=1.3.2  # [py<37]
```

Comment:
I think it'd be more intuitive to write `>=1.3.2,<=1.5.4`

Reply:
Fixed.